### PR TITLE
Add Trino 413 release notes

### LINKED ISSUE #16895: Release notes for 413

### PR DESCRIPTION
**Pull request by colebow**
## Description

Assemble the release notes for the upcoming Trino 413 release. Fixes #16895 

## Release notes

(x) This is not user-visible or docs only and no release notes are required.

## Verification for each pull request

Format: maintainer, PR/issue number, ✅ / ❌ rn ✅ / ❌ docs
✅ rn - release note added and verified, or assessed to be not necessary, set to ❌ rn before completion
✅ docs - need for docs assessed and merged, or assessed to be not necessary, set to ❌ docs before completion

## 5 Apr 2023

* #16896 ✅ rn ✅ docs
* #16808 ✅ rn ✅ docs
* #16880 ✅ rn ✅ docs
* #16826 ✅ rn ✅ docs
* #16485 ✅ rn ✅ docs
* #16899 ✅ rn ❌ docs
* #16693 ✅ rn ❌ docs
* #16849 ✅ rn ✅ docs

## 6 Apr 2023

* #16890 ✅ rn ✅ docs
* #16902 ✅ rn ✅ docs
* #16870 ✅ rn ✅ docs
* #15378 ✅ rn ✅ docs
* #16918 ✅ rn ✅ docs
* #16124 ✅ rn ✅ docs
* #16901 ✅ rn ✅ docs

## 7 Apr 2023

* #16906 ✅ rn ✅ docs
* #16811 ✅ rn ✅ docs
* #16914 ✅ rn ✅ docs
* #16842 ✅ rn ✅ docs
* #16748 ✅ rn ✅ docs

## 8 Apr 2023

* #16935 ✅ rn ✅ docs

## 10 Apr 2023

* #16943 ✅ rn ✅ docs
* #16861 ✅ rn ✅ docs
* #16939 ✅ rn ✅ docs
* #16862 ✅ rn ✅ docs
* #16936 ✅ rn ✅ docs
* #16917 ✅ rn ✅ docs
* #16856 ✅ rn ✅ docs
* #16792 ✅ rn ✅ docs
* #16501 ✅ rn ✅ docs
* #16937 ✅ rn ✅ docs
* #16753 ✅ rn ✅ docs
* #16938 ✅ rn ✅ docs

## 11 Apr 2023

* #16930 ✅ rn ✅ docs
* #16889 ✅ rn ✅ docs
* #16956 ✅ rn ✅ docs
* #16714 ✅ rn ✅ docs
* #16958 ✅ rn ✅ docs
* #16952 ✅ rn ✅ docs
* #16955 ✅ rn ✅ docs
* #16932 ✅ rn ✅ docs
* #16953 ✅ rn ✅ docs
* #16957 ✅ rn ✅ docs

## 12 Apr 2023

* #16961 ✅ rn ✅ docs
* #16963 ✅ rn ✅ docs
* #16977 ✅ rn ✅ docs
* #16973 ✅ rn ✅ docs
* #16970 ✅ rn ✅ docs
* #16945 ✅ rn ✅ docs

### CODE REVIEW

File: docs/src/main/sphinx/release/release-413.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Release 413 (12 Apr 2023)
+
+## General
+
+* Improve performance of queries involving window operations or
+  [row pattern recognition](/sql/pattern-recognition-in-window) on small
+  partitions. ({issue}`16748`)
+* Improve performance of queries with `TopN` operations. ({issue}`16753`)
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-413.md:8]: TopN ... isnt really an operation .. what do we call this in other entries?

File: docs/src/main/sphinx/release/release-413.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+* Disallow enabling change data feed when the table contains
+  `_change_type`, `_commit_version` and `_commit_timestamp` columns. ({issue}`16913`)
+* Fix incorrect results when reading `INT32` values without a decimal logical
+  annotation in Parquet files. ({issue}`16938`)
+
+## Hive connector
+
+* Fix incorrect results when reading `INT32` values without a decimal logical
+  annotation in Parquet files. ({issue}`16938`)
+* Fix incorrect results when the file path contains hidden characters. ({issue}`16386`)
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-413.md:28]: is that for specific reader (like Parquet or ORC) or in general?

File: docs/src/main/sphinx/release/release-413.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+## Iceberg connector
+
+* Fix incorrect results when reading `INT32` values without a decimal logical
+  annotation in Parquet files. ({issue}`16938`)
+* Fix failure when creating a schema with a username containing uppercase
+  characters in the Iceberg Glue catalog. ({issue}`16116`)
+
+## Oracle connector
+
+* Add support for [table comments](/sql/comment) and creating tables with
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-413.md:44]: They are kind of the same thing right? Would you ever implement one without the other?

File: docs/src/main/sphinx/release/release-413.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  comments. ({issue}`16898`)
+
+## Phoenix connector
+
+* Add support for {doc}`/sql/merge`. ({issue}`16661`)
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-413.md:49]: NICE!

**Review by mosabua — CHANGES_REQUESTED** [review]

File: docs/src/main/sphinx/release/release-413.md (hunk 3 above)
Comment (colebow) [reply docs/src/main/sphinx/release/release-413.md:44]: Yeah, we've done it a lot, actually.

File: docs/src/main/sphinx/release/release-413.md (hunk 1 above)
Comment (martint) [reply docs/src/main/sphinx/release/release-413.md:8]: In this case, it's queries involving `row_number()` and `rank()` window functions.

File: docs/src/main/sphinx/release/release-413.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
+# Release 413 (12 Apr 2023)
+
+## General
+
+* Improve performance of queries involving window operations or
+  [row pattern recognition](/sql/pattern-recognition-in-window) on small
+  partitions. ({issue}`16748`)
+* Improve performance of queries with `TopN` operations. ({issue}`16753`)
+* Fix potential query failure when collecting optimizer rule execution
```
Comment (martint) [inline docs/src/main/sphinx/release/release-413.md:9]: I don't think this is accurate. The failure happens when cancelling a query, but I'm not sure whether it's user-visible or how does it manifest to the user. cc @findepi ?

File: docs/src/main/sphinx/release/release-413.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+  statistics. ({issue}`16960`)
+
+## Delta Lake connector
+
+* Add support for writing the `timestamp with time zone` type in
```
Comment (martint) [inline docs/src/main/sphinx/release/release-413.md:14]: Add support for nested `timestamp with time zone` values in structural data types.

**Review by mosabua — APPROVED** [review]
Looks good now. Thanks.

**Pull request merged**